hadoop-ozone/recon-codegen/src/main/java/org/hadoop/ozone/recon/schema/SchemaVersionTableDefinition.java

```diff
@@ -66,4 +68,16 @@ private void createSchemaVersionTable() throws SQLException {
         .execute();
   }
 
+  /**
+   * Insert the version of the MLV for the table if it doesn't already exist.
+   * @param version The version value to be inserted
+   * @throws SQLException If the insert operation failed
+   */
+  public void insertCurrentVersion(int version) throws SQLException {
+    if (!CHECK_COLUMN_HAS_VALUE.apply(conn, SCHEMA_VERSION_TABLE_NAME, "version_number")) {
```

Comment:
if already have value, do this need make insertion fail, right? or some action required as version may not be one as expected.